pom.xml

```
@@ -77,22 +77,22 @@
 
     <properties>
         <!-- Minimal Jenkins version (2.14.0 ) required by for pipeline job -->
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.289.3</jenkins.version>
```

Usually we prefer the .1 release of the LTS.
```suggestion
        <jenkins.version>2.289.1</jenkins.version>
```